### PR TITLE
Tracking momentum change

### PR DESCRIPTION
**Pull request by tlprotzman**
Draw plots as a function of momentum in addition to transverse momentum, getting out of mid rapidity habits where p ~= pT ~= E.

@raymondEhlers

### CODE REVIEW

File: treeAnalysis/jetresolutionhistos.cxx [hunk 1]
```diff
@@ -73,6 +81,8 @@ TH2F *h2D_truth_eta_phi[njettypes] = {NULL};
 TH2F *h2D_cluster_eta_phi[njettypes] = {NULL};
 
 
+double momentum(fastjet::PseudoJet jet) {return TMath::Sqrt(jet.px() * jet.px() + jet.py() * jet.py() + jet.pz() * jet.pz());}
```
Comment (raymondEhlers) [inline treeAnalysis/jetresolutionhistos.cxx:84]: Just fyi - better to use `std::sqrt` . Many TMath functions are basically around for historical reasons

**Review by raymondEhlers — APPROVED** [review]

File: treeAnalysis/jetresolutionhistos.cxx (hunk 1 above)
Comment (tlprotzman) [reply treeAnalysis/jetresolutionhistos.cxx:84]: Good to know, thanks.  If you see stuff like this in the future feel free to hold off on merging it, I'd like the use best practices where I can.

Comment (raymondEhlers) [reply treeAnalysis/jetresolutionhistos.cxx:84]: Sure, will do - I've been merging quickly for the sake of moving quickly, but I can move a bit more deliberately

Comment (raymondEhlers) [reply treeAnalysis/jetresolutionhistos.cxx:84]: I was a little distracted earlier - the better answer is that you don't need this function - just use `PseudoJet::modp()`

Comment (tlprotzman) [reply treeAnalysis/jetresolutionhistos.cxx:84]: Ah perfect, somehow when I looked over the fastjet documentation I missed that!

**Pull request merged**